conda/testing/integration.py

```diff
@@ -301,6 +301,8 @@ def run_command(command, prefix, *arguments, **kwargs):
         reset_context([os.path.join(prefix + os.sep, "condarc")], args)
     return stdout, stderr, result
 
+import tempfile
+req_tempdir = tempfile.TemporaryDirectory().__enter__()  # type: ignore
```

Comment:
could go into memory and write to a tempfile on create, or pytest could pass in a cache object